tests/utils/session.py

```diff
@@ -94,6 +95,29 @@ def _get_response(self):
 
         return self.responses[0]
 
+    @staticmethod
+    def cursor_paged_resource(base_method: Callable[..., dict], path: str,
```

Comment:
Nice!

Reply:
Thank Aris - I just copied this to the test code